Move prover returns: exiting with model building errors
error: no function named `predicate` found
   ┌─ tests/sources/functional/inline_fun.move:12:46
   │
12 │                 invariant forall k in 0..i: !predicate(v[k]);
   │                                              ^^^^^^^^^^^^^^^

error: [internal] bytecode does not match AST: `test_filter_predicate_0` in bytecode but not in AST (available in AST: 0x1::vector::contains, 0x1::vector::index_of, 0x1::vector::append, 0x1::vector::borrow, 0x1::vector::borrow_mut, 0x1::vector::destroy_empty, 0x1::vector::empty, 0x1::vector::filter, 0x1::vector::fold, 0x1::vector::for_each, 0x1::vector::for_each_mut, 0x1::vector::for_each_ref, 0x1::vector::is_empty, 0x1::vector::length, 0x1::vector::map, 0x1::vector::pop_back, 0x1::vector::push_back, 0x1::vector::remove, 0x1::vector::reverse, 0x1::vector::singleton, 0x1::vector::swap, 0x1::vector::swap_remove, 0x42::Test::filter, 0x42::Test::test_filter)
  ┌─ <internal>:1:1
  │
1 │ <internal>
  │ ^^^^^^^^^^
